packages/badge/src/Default/Default.tsx

```diff
@@ -25,21 +25,14 @@ export const Component = (props: BadgeDefaultProps) => {
         ],
       },
     },
-    ['prefix', 'theme', 'size']
+    ['prefix', 'theme']
   );
 
   themePrefixes[baseClass] = `owlui-${baseClass}`;
 
-  // below isn't great, but passing 'pill' to the Badge is not generating the class 'rounded-pill' on the <span> like it should be. Ideally we could get that working and then add 'rounded-pill' to the prefix ignore list
-  if (pill) {
-    locals['className'] += ' owlui-rounded-pill';
```

Comment:
We can consider the `rounded-pill` a utility class, so let's add it to the postcss config.